src/components/input/input.tsx

```diff
@@ -152,46 +172,70 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
   }
 
   /**
-   * Maximum length of text input.
+   * Specifies the maximum length of text for the component's value.
    *
    * @deprecated use maxLength instead
    */
   @Prop({ reflect: true }) maxlength?: number;
 
-  /** Maximum length of the input value */
+  /**
+   * Specifies the maximum length of text for the component's value.
+   *
+   * @mdn [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#maxlength)
+   */
   @Prop({ reflect: true }) maxLength?: number;
 
-  /** Minimum length of the text input */
+  /**
+   * Specifies the minimum length of text for the component's value.
+   *
+   * @mdn [minlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#minlength)
+   */
   @Prop({ reflect: true }) minLength?: number;
 
-  /** The name of the input */
+  /**
+   * Specifies the name of the component.
+   *
+   * @mdn [name](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name)
+   */
   @Prop({ reflect: true }) name: string;
 
-  /** specify the placement of the number buttons */
+  /** Specifies the placement of the buttons for type "number". */
   @Prop({ reflect: true }) numberButtonType?: InputPlacement = "vertical";
 
-  /** explicitly whitelist placeholder attribute */
+  /**
+   * Specifies placeholder text for the component.
+   *
+   * @mdn [placeholder](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder)
+   */
   @Prop() placeholder: string;
 
-  /** optionally add prefix  */
+  /** Adds text to the start of the component.  */
   @Prop() prefixText?: string;
 
-  /** When true, a field cannot be modified. */
+  /**
+   * When true, the field cannot be modified.
+   *
+   * @mdn [readOnly](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#readonly)
+   */
   @Prop() readOnly = false;
 
-  /** is the input required */
+  /** When true, the component must have a value in order for the form to submit. */
   @Prop() required = false;
 
-  /** specify the scale of the input, defaults to m */
+  /** Specifies the size of the component. */
   @Prop({ mutable: true, reflect: true }) scale: Scale = "m";
 
-  /** specify the status of the input field, determines message and icons */
+  /** Specifies the status of the input field, which determines message and icons. */
   @Prop({ mutable: true, reflect: true }) status: Status = "idle";
 
-  /** input step */
+  /**
+   * Specifies the granularity that the component's value must adhere to.
+   *
+   * @mdn [step](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#step)
```

Comment:
The MDN url with `../#step` doesn't seem to navigate to the placement in the document. It seems that page doesn't have the capability for the attributes. 

Would `https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step` suffice in its place?

Reply:
Weird, it works for me. But sure I can change it